CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### Changed - PRC, 2022-01-19
```

Comment:
@pcolarco You don't need initials or anything here. Git and GitHub can tell us who changed a line of code.

Also, this should go under the `## [Unreleased]` section of the changelog.

Would you like me to fix this up for you? GitHub is showing a conflict anyway in the changelog that needs to be resolved. (I think your local `develop` might be a bit out of date?)

Reply:
@mathomp4 Could you please fix this?
I got with @mmanyin to commit the changes to my branch. We tried to go further and issue a pull request, but you are correct I'm on an older model (10.17.6). I have no idea how to merge this into a newer model and thought I was waiting for someone to bless GOCART2G in the newer model but made this fix anyway. Sorry, don't really know what I'm doing here.

Reply:
Oh no, it should work. Let me try...